InvisibleMapCreator2/State Machine/AppState.swift

```diff
@@ -93,20 +99,28 @@ enum RecordMapState: StateType {
     // In response to an event, RecordMapState may emit a command
     mutating func handleEvent(event: Event) -> [Command] {
         switch (self, event) {
-        case(.RecordMap, .NewARFrame(let cameraFrame)):
-            return [.RecordData(cameraFrame: cameraFrame)]
-        case(.RecordMap, .NewTagFound(let tag, let cameraTransform, let snapTagsToVertical)):
-            return [.DetectTag(tag: tag, cameraTransform: cameraTransform, snapTagsToVertical: snapTagsToVertical), .EnableAddLocation]
-        case(.RecordMap, .SaveLocationRequested(let locationName)):
-            return [.PinLocation(locationName: locationName)]
-        case(.RecordMap, .ViewLocationsRequested):
-            self = .ViewLocations
-            return []
-        case(.RecordMap, .DismissLocationsRequested):
-            self = .RecordMap
-            return []
-            
-        default: break
+            case(.RecordMap, .NewARFrame(let cameraFrame)):
+                return [.RecordData(cameraFrame: cameraFrame), .UpdateInstructionText]
+            case(.RecordMap, .NewTagFound(let tag, let cameraTransform, let snapTagsToVertical)):
+                return [.DetectTag(tag: tag, cameraTransform: cameraTransform, snapTagsToVertical: snapTagsToVertical)]
+            case(.RecordMap, .PlanesUpdated(let planes)):
+                return [.UpdatePlanes(planes: planes)]
+            // case(.RecordMap, .PauseRecordingPressed):
```

Comment:
explain why this is commented out or get rid of it.  E.g., "// uncomment this to implement case XYZ").